test/onnx/test_pytorch_onnx_no_runtime.py

```diff
@@ -107,8 +155,427 @@ def forward(self, y: Optional[Tensor]) -> Optional[Tensor]:
         )
 
 
-instantiate_parametrized_tests(TestOptionalOutput)
+class TestONNXExport(common_utils.TestCase):
```

Comment:
In this case, common_utils.Testcase is used as assertAtenOp() is its member function, and we use that to check Aten fallback.